MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -131,6 +131,8 @@ protected virtual string GetOutputFileExtension(StreamState state)
         /// </summary>
         private string GetOutputFilePath(StreamState state, EncodingOptions encodingOptions, string outputFileExtension)
         {
+            if (outputFileExtension == null) outputFileExtension = "";
```

Comment:
It would be better to change line 139 to this instead of assigning an empty string here

```cs
var ext = outputFileExtension?.ToLowerInvariant();
```

Reply:
Sure makes sense - forgot that operator existed in C# :)